### PR TITLE
Add minimal documentation for tagging

### PR DESCRIPTION
**Pull request by giorgiosironi**
One point of https://github.com/libero/libero/issues/166

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 An automated deployment process for several Libero standard projects, targeting a single VM [infrastructure](https://github.com/libero/infrastructure) at first, instantiated in multiple environments.
 
+## Deployment
+
+To:
+
+- `unstable`: every `master` build of this repository deploys the newest versions of the projects.
+- `demo`: pushing a new `latest/*` tag (e.g. `latest/20190701`) deploys the newest versions of the projects.
```
Comment (GiancarloFusiello) [inline README.md]: Where should someone push latest to?

Comment (giorgiosironi) [reply README.md]: Github to the `libero/environments` repository? Will make it clear.

File: README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ An automated deployment process for several Libero standard projects, targeting
 To:
 
 - `unstable`: every `master` build of this repository deploys the newest versions of the projects.
-- `demo`: pushing a new `latest/*` tag (e.g. `latest/20190701`) deploys the newest versions of the projects.
+- `demo`: pushing a new `latest/*` tag (e.g. `latest/20190701`) to the `libero/environments` Github repository deploys the newest versions of the projects.
```
Comment (GiancarloFusiello) [inline README.md]: Is this referring to a new git tag?

Comment (giorgiosironi) [reply README.md]: yes, not a Docker image tag if that's the source of the confusion

File: README.md [hunk 3]
```diff
@@ -2,6 +2,13 @@
 
 An automated deployment process for several Libero standard projects, targeting a single VM [infrastructure](https://github.com/libero/infrastructure) at first, instantiated in multiple environments.
 
+## Deployment
+
+To:
+
+- `unstable`: every `master` build of this repository deploys the newest versions of the projects.
+- `demo`: pushing a new `latest/*` tag (e.g. `latest/20190701`) to the `libero/environments` Github repository deploys the newest versions of the projects.
```
Comment (GiancarloFusiello) [inline README.md]:
```suggestion
- `demo`: pushing a new git tag to the `libero/environments` Github repository using the prefix `latest/` deploys the latest docker images to the demo environment.
```

**Review by GiancarloFusiello — APPROVED** [review]

**Pull request merged**